docs/Friendly_User/Applications/amrwind.md

```diff
@@ -0,0 +1,165 @@
+AMR-Wind
+========
+
+## AMR-Wind on Kestrel
+
+AMR-Wind is a massively parallel, block-structured adaptive-mesh,
+incompressible flow solver for wind turbine and wind farm
+simulations. The primary applications for AMR-Wind are: performing
+large-eddy simulations (LES) of atmospheric boundary layer (ABL)
+flows, simulating wind farm turbine-wake interactions using actuator
+disk or actuator line models for turbines, and as a background solver
+when coupled with a near-body solver (e.g., Nalu-Wind) with overset
+methodology to perform blade-resolved simulations of multiple wind
+turbines within a wind farm. For more information see
+https://github.com/Exawind/amr-wind
+
+Installation of AMR-Wind on GPU nodes
+-------------------------------------
+
+
+AMR-wind can be installed by following the instructions [Here](https://exawind.github.io/amr-wind/user/build.html#building-from-source) .
+ON Kestrel GPU nodes, this can be achieved by first loading the following modules:
+
+```
+module restore 
+source /nopt/nrel/apps/gpu_stack/env_cpe23.sh
+ml gcc
+ml PrgEnv-nvhpc
+ml nvhpc/24.1
+ml cray-libsci/22.12.1.1
+ml cmake/3.27.9
+ml python/3.9.13
+```
+
+Make sure the following modules are loaded using module list.
+
+
+  1) libfabric/1.15.2.0
```

Comment:
For list formatting, use "." instead of ")" and no extra space in between lines. For example:
1.
2.